src/client/store/actions.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Internal dependencies
+ */
+import {
+	REGISTER_FEATURE,
+	RECEIVE_FEATURE,
+	UNREGISTER_FEATURE,
+	RECEIVE_FEATURES,
+	REGISTER_FEATURE_CALLBACK,
+} from './constants';
+
+import { store } from './index';
+
+// Action Creators
+export function registerFeature( feature ) {
```

Comment:
since we're now using typescript, can we add the feature type annotation where appropriate